### DISCUSSION TITLE
curious about a use-case for this project other than stalking and targeted harassment?

### DISCUSSION THREAD

**Discussion by chee** [sherlock-project/sherlock#986] [Q&A]
what are we all using it for, other than as a stalking and harassment tool?

**Comment by hoadlck** [comment]
I always thought the purpose was to find available usernames so one could have a consistent brand across all of their social media accounts.

But, it does make me wonder...I have never understood why Sherlock is so popular.  Is this what this tool is really used for?

I wish there was some way to keep the tool from being used for bad purposes.  But, I suspect I was being overly naive thinking it was just used for branding.

**Reply by ghost** [reply]
Most likely. A lot of technologists want to solve an immediate problem without understanding the scope of the use of their tools in the long term.

**Comment by SethFalco** [comment]
I use it for the same reason as this person:
* https://github.com/sherlock-project/sherlock/discussions/945

To find old accounts or things I forgot about, and to delete them.
Some companies are pretty nice about it too like MySpace, Yahoo Answers, or Newgrounds, with no access to the account or email if you can just give some context around the account and say it hasn't been active for 10+ years they're happy to delete it.

I also show others how to use it and let them collate a list of all their old accounts, so they can delete theirs too, which helps them feel a lot better about their digital presence.

The reality is, while this tool has the capabilities for stalking, you should consider anything you post on the internet public the moment you push send. And in some cases even after you delete it… because some sites don't actually delete data but rather just mark it as deleted/hidden in their database, and it'll still come out later anyway when or if a breach ever occurs.

This is a great tool for helping you identify all that data you've left online for people to find in the first place so you can get rid of it. (Or at least the things you regret putting online or don't want others finding.)

TL;DR: All good things can be used for evil purposes. In my opinion, the pros outweigh the cons here, we should continue to make it easier for people to find and delete their accounts/data on the internet.
